prime-angular-frontend/src/app/modules/enrolment/pages/next-steps/next-steps.component.ts

```diff
@@ -171,38 +172,51 @@ export class NextStepsComponent extends BaseEnrolmentProfilePage implements OnIn
   }
 
   public sendProvisionerAccessLink() {
-    const data: DialogOptions = {
-      title: 'Confirm Email',
-      message: `Are you sure you want to send your Approval Notification?`,
-      actionText: 'Send',
-    };
-    this.busy = this.dialog.open(ConfirmDialogComponent, { data })
-      .afterClosed()
-      .pipe(
-        exhaustMap((result: boolean) => {
-          if (result) {
-            this.complete = true;
-
-            let emailPairs = this.careSettingConfigs.map((config) => {
-              return {
-                emails: config.formArray.value.map(email => email.email),
-                careSettingCode: config.settingCode,
-                healthAuthorityCode: config.settingCode === CareSettingEnum.HEALTH_AUTHORITY ? config.healthAuthorityCode : null,
-              }
-            });
 
-            return this.enrolmentResource.sendProvisionerAccessLink(emailPairs, this.enrolment.id);
-          } else {
-            return EMPTY;
-          }
-        })
-      )
-      .subscribe(() => {
-        this.toastService.openSuccessToast('Email was successfully sent');
-        this.router.navigate([EnrolmentRoutes.PHARMANET_ENROLMENT_SUMMARY],
-          { relativeTo: this.route.parent, queryParams: { initialEnrolment: this.initialEnrolment } });
-      });
-    this.onPageChange({ atEnd: true });
+    if (!this.atLeastOneEmailFilled()) {
+      const data: DialogOptions = {
+        title: 'Missing Email',
+        message: `Please enter at least one email for the Approval Notification?`,
```

Comment:
I think question mark at end of sentence is unnecessary and icon should be "warning" or "error"

Reply:
Update as suggested.